### ISSUE TITLE
How to use DevTools

### ISSUE THREAD

**Issue by carolinarjona** [anseki/leader-line#318]
Hello!

I have a problem. The lines only appear if I move the window and the scroll does not work correctly. I show you in this gif:
![ScrollAndDom](https://user-images.githubusercontent.com/79989429/165047395-74ec6279-a470-4095-b3b8-e0348726c4b0.gif)

Thank you so much!

**Comment by anseki** [comment]
Hi @carolinarjona, thank you for the comment.
Nobody can find bugs in your code without reading the code, then I infer it.
Maybe you did make the lines before the elements are not rendered (positioned) yet.

Without reference to LeaderLine, when you develop something with web browsers, you should use DevTools of web browsers.
You can use it for checks above also. For example, write `debugger` statement at a breakpoint that you want to check.
```js
debugger;
const line = new LeaderLine(document.querySelector('.card-1'), document.querySelector('.card-2'));
```
If you are using modern browser, the running is paused at the break point, and you can check the elements and you can do more in DevTools of your browser.
For example, input `document.querySelector('.card-1')` into "Console" (*not `console.log()`*), and push Enter key.

For test, wrap the code with `setTimeout`, for example, replace the code `new LeaderLine(...)` with `setTimeout(() => { new LeaderLine(...); }, 400)`.
Also, it seems that the lines were added to another window (frame) from the window that has the elements.

**Comment by carolinarjona** [comment]
Thanks!

I don't realize than I was using the 'leader-line-new' library to import easily in Angular (because I can't use the 'LeaderLine.positionByWindowResize = false' if I use 'leader-line-new' 🤪) and that's the problem buuuuut if I delete it and I use only the 'leader-line' I have the **LeaderLine is not defined** problem. I did it in Stackblitz (https://stackblitz.com/edit/angular-ivy-zsiwjm) and works fine but I can't recreate in my business project and I did the same but doesn't work 😢

1. Install library
2. Import the script in the index.html and in the angular.json
3. Declare var LeaderLine: any.
